content/en/docs/languages/cpp/library.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Using instrumentation libraries
+linkTitle: Libraries
+weight: 40
+---
+
+{{% docs/languages/libraries-intro "C++" %}}
+
+## Using instrumentation libraries
+
+When you develop an app, you might use third-party libraries and frameworks to
+accelerate your work. If you then instrument your app using OpenTelemetry, you
+might want to avoid spending additional time to manually add traces, logs, and
+metrics to the third-party libraries and frameworks you use.
+
+Many libraries and frameworks already support OpenTelemetry or are supported
+through OpenTelemetry
+[instrumentation](https://opentelemetry.io/docs/concepts/instrumentation/libraries/),
+so that they can generate telemetry you can export to an observability backend.
+
+If you are instrumenting an app or service that use third-party libraries or
+frameworks, follow these instructions to learn how to use natively instrumented
+libraries and instrumentation libraries for your dependencies.
+
+## Use natively instrumented libraries
+
+If a library comes with OpenTelemetry support by default, you can get traces,
+metrics, and logs emitted from that library by adding and setting up the
+OpenTelemetry SDK with your app.
+
+The library might require some additional configuration for the instrumentation.
+See the documentation for that library to learn more.
+
+If a library doesn't include OpenTelemetry support, you can use
+[instrumentation libraries](https://opentelemetry.io/docs/specs/otel/glossary/#instrumentation-library)
+to generate telemetry data for a library or framework.
+
+For example,
+[NGINX instrumentation](https://github.com/open-telemetry/opentelemetry-cpp-contrib/tree/main/instrumentation/nginx)
+adds OpenTelemetry distributed tracing support to NGINX.
```

Comment:
The NGINX example is a complicated one, since there are 3 different modules, and it's also not an instrumentation library in that sense. I lean towards not including this section. Instead we could add a note/warning that we are looking for people implementing an instrumentation library.

```suggestion
```

Reply:
should i write note/warning and then commit?

Reply:
We can have it as a follow up, as you like.